src/modules/Home/Home.js

```diff
@@ -1,23 +1,59 @@
 import React from 'react';
-import AppLogo from '../../assets/applogo.svg';
+import {useState} from 'react';
+import './Home.css';
 
 function Home() {
+  const API_KEY = process.env.REACT_APP_NAPSTER_API_KEY;
+  const [genreData, setGenreData] = useState([]);
```

Comment:
Why are you using useState()? I think it's for instances in which the browser needs to update the page more than once. For example, a counter that increments every minute.

Reply:
Actually, maybe that makes sense to use so the user doesn't have to do the same api call again after changing pages.

Reply:
Since the API call is asynchronous, the return() will most likely be called before the genre data is retrieved. As a result,  if I do not update the component one more time by updating `genreData` with `useState` once the API call finishes, the genre images will not be displayed.